lang/semgrep-grammars/src/semgrep-kotlin/src/scanner.c

```diff
@@ -213,11 +213,11 @@ bool scan_multiline_comment(TSLexer *lexer) {
         if (after_star) {
           after_star = false;
           nesting_depth--;
-          // if (nesting_depth == 0) {
+          if (nesting_depth == 0) {
```

Comment:
what are those changes?

Reply:
these are ported changes from the `scanner.c` in `tree-sitter-kotlin`. we have to manually update it because we keep a corresponding version, but with some minor semgrep-specific changes. these correspond to the scanner changes i made in https://github.com/returntocorp/tree-sitter-kotlin/commit/e3bbe6080df359a60c91f4d2645e3510cc6343fe to the normal `tree-sitter-kotlin`

Reply:
Ah ok. I hope the semgrep-specific stuff are clearly tagged with a comment.